File: Java/set2/Q6.java
// 6. Write a program to find biggest of three numbers without using logical operators by taking input from the user. 

class Q6
{
    public static void main(String[]args)
    {
        int a=6, b=3, c=0;
        if(a>b)
        {
            if(a>c)
                System.out.println(a+" is bigger");
            else
                System.out.println(c+" is bigger");
        }
        else if(b>a)
        {
            if(b>c)
                System.out.println(b+" is bigger");
            else
                System.out.println(c+ " is bigger");
        }
    }
}

File: Python/set2/Q6.py
# 6. Write a program to find the area of triangle when three sides are given. (Hint: use heron’s equation  

import math

a,b,c=6,5,4
s=(a+b+c)/2
a=math.sqrt(s*(s-a)*(s-b)*(s-c))
print("Area:",a)